README.md

```diff
@@ -16,14 +16,29 @@ The STAC Generator accepts a JSON payload with the following structure:
     "https://path-to-cloud-storage/shapefile.shp",
     "https://path-to-cloud-storage/first-file.tif",
     "https://path-to-cloud-storage/second-file.tif"
-  ],
-  "metadata": ["https://path-to-cloud-storage/metadata.json"],
-  "method": "POST | PUT | DELETE"
+  ]
 }
 ```
-## Optional Payload
 
-`metadata` and `method` do nothing for now. This will change.
+## Environment Variables
+
+This application is configured using the following environment variables:
+
+### Redis Configuration
+
+- `REDIS_HOST`: This is the hostname of your Redis instance. The default is `redis`.
+- `REDIS_PORT`: This is the port number of your Redis instance. The default is `6379`.
+- `REDIS_INCOMING_LIST_NAME`: This is the name of the Redis list that the application will monitor for incoming tasks. The default is `stac_generator_generate`.
```

Comment:
these are named REDIS_INPUT_LIST_NAME and REDIS_OUTPUT_LIST_NAME in the previous microservice. But not a big issue, can leave it as is. It is clear enough.

But is REDIS_OUTGOING_LIST_USED?